cdap-data-fabric/src/test/java/co/cask/cdap/api/dataset/lib/IndexedTableTest.java

```diff
@@ -688,4 +689,38 @@ private Row readFirst(Scanner scanner) {
     }
     return row;
   }
+
+  private void verifyRow(Row result, byte[] expectedRow, byte[][] columns, byte[][] expected) {
```

Comment:
The other TableTest has a lot of these methods as well.  Should we move them to a common TableAssert class so that static methods can be reused where needed?